### PR TITLE
Add Developers for Firefox landing page (Fixes #6178)

### LINKED ISSUE #6178: [10/5] Developers for Firefox - /new variation landing page
Description: create a /new variation landing page to support the  ['Developers for Firefox' program](https://docs.google.com/document/d/1t6YWu8rvUOgzahv9uZPM_N6fxnrGWhF0m7Phedo_hKc/edit). In this mktg program, developers are encouraged to promote use of Firefox, posting images with a URL (link with UTM parameters to enable anonymous tracking). Users clicking the link are taken to this  landing page with copy inline with their 'referral journey'

Success Criteria: 
- [ ] We track the number of installs / aDAU in aggregate for the whole campaign with the single referral URL (Traffic to landing page attributed using source tags)
- [ ] aDAU attributed to traffic in Telemetry
- [ ] URL:  /firefox/this-browser-comes-highly-recommended
- [ ] Set to noindex (not be indexed by search engines)

[Copy doc](https://docs.google.com/document/d/1dR93_giMMpa77SVkanE9x-kkTTMpE1kmtXXejtdYs5I/edit)

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds a new landing page at `/firefox/firefox/this-browser-comes-highly-recommended/`

## Issue / Bugzilla link
#6178

## Testing
Demo: https://bedrock-demo-agibson.oregon-b.moz.works/en-US/firefox/this-browser-comes-highly-recommended/

### CODE REVIEW

**Comment by alexgibson** [comment]
Ok this is now ready for review.

File: bedrock/firefox/templates/firefox/recommended.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "base-protocol.html" %}
+
+{# "noindex" pages should not have the canonical or hreflang tags: bug 1442331 #}
+{% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
+
+{% block page_title %}{{ _('Fast. Private. Fearless. Firefox fights for you.') }}{% endblock %}
+{% block page_desc %}{{ _('This browser comes highly recommended. ') }}{% endblock %}
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/recommended.html]: Nit - extra space before end quote.

File: bedrock/firefox/templates/firefox/recommended.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            <p class="mzp-c-card-picto-desc">{{_('Truly Private Browsing with Tracking Protection') }}</p>
+          </div>
+        </section>
+      </li>
+    </ul>
+  </div>
+
+  <aside class="c-aside">
+    <div class="mzp-l-content">
+      <p><a href="https://mozilla.github.io/for-firefox/" data-link-type="link" data-link-name="Want a Firefox badge of your own?">{{ _('Want a Firefox badge of your own?') }}</a></p>
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/recommended.html]: Could add a `rel="external"` if we're still doing that kind of thing.

File: bedrock/firefox/templates/firefox/recommended.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    </div>
+  </aside>
+
+  <section class="secondary-cta mzp-c-call-out-compact mzp-t-firefox mzp-t-product-firefox">
+      <div class="mzp-l-content">
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/recommended.html]: Nit - indentation.

File: bedrock/firefox/templates/firefox/recommended.html [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        </div>
+        <div class="mzp-c-call-out-cta">
+          <div class="mzp-c-call-out-cta-container">
+            <div class="mzp-c-button-download">
+              {{ download_firefox(alt_copy=_('Download Firefox'), download_location='secondary cta', button_color='mzp-c-button mzp-t-download') }}
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/recommended.html]: Maybe center this content for `< 768px` so it matches the rest of the page?

Comment (jpetto) [reply bedrock/firefox/templates/firefox/recommended.html]: This content is also a little misaligned in IE 10 & 11. (But maybe not enough to worry too much about?)

screen shot 2018-10-05 at 10 25 48 am

Comment (alexgibson) [reply bedrock/firefox/templates/firefox/recommended.html]: This is a protocol component / bug, which is being tracked upstream https://github.com/mozilla/protocol/issues/218, so not fixing here.

**Comment by alexgibson** [comment]
> Also, the destination URL for "What a Firefox badge of your own?" does not look ready. Not sure who's responsible for that, but IMO it should be fixed before launching this page.

The campaign isn't launching until the end of next week, so we're fine to merge & push this ahead of time.

**Comment by alexgibson** [comment]
@jpetto nits picked, image smushed. Thanks for the review 👍

**Pull request merged**